src/Form/CaseEventNewType.php

```diff
@@ -53,13 +53,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'expanded' => true,
                 'required' => false,
             ])
+            ->add('manualSenders', TextType::class, [
+                'label' => $this->translator->trans('Senders', [], 'case_event'),
+                'required' => false,
+            ])
             ->add('recipients', ChoiceType::class, [
                 'label' => $this->translator->trans('Recipients', [], 'case_event'),
                 'choices' => $choices,
                 'multiple' => true,
                 'expanded' => true,
                 'required' => false,
             ])
+            ->add('manualRecipients', TextType::class, [
+                'label' => $this->translator->trans('Senders', [], 'case_event'),
+                'required' => false,
+            ])
```

Comment:
Consider changing “manual” to “additional” and use `TextArea` with one “additional” per line.

Adding a `TextHelper` with an `getLines` function splitting a multiline text into non-empty string values will be handy here. And in the other places where we do this.